core/include/drivers/tzc380.h

```diff
@@ -202,6 +202,10 @@ enum tzc_action {
 #define TZC_ATTR_REGION_ENABLE	0x1
 #define TZC_ATTR_REGION_DISABLE	0x0
 
+#define LOCKDOWN_RANGE_ENABLE	BIT(31)
```

Comment:
use tabs for indentation, here and below.

Reply:
This should be a tab, marking with the mouse identifies it as one. I have however aligned this line with the next line.